perf/perf-producer.go

```diff
@@ -22,11 +22,12 @@ package main
 import (
 	"context"
 	"encoding/json"
+	`github.com/Anankke/bilibili-backup/library/net/http/blademaster/middleware/rate`
```

Comment:
Is this import intended?

Reply:
oh no, actually, we shouldn't import this, next pull request, i will remove it

Reply:
No worries. I already removed it in my code.